client/my-sites/importer/style.scss

```diff
@@ -219,16 +219,19 @@
 		}
 	}
 
-
-
 	& .site-importer__site-importer-confirm-site-pane-container {
 		display: flex;
-		flex-direction: row;
 		align-content: center;
-		justify-content: flex-end;
+		justify-content: center;
```

Comment:
Can we hide the `Is this your site?` prompt & buttons (`.site-importer__site-importer-confirm-site-pane-container`) until the image is fully loaded (or the Promise rejects)?

Not sure if the wifi is slow where I am or if it took a bit to process, but I saw a screen that looked like this for 5ish seconds the first few times I tried this:

<img width="729" alt="screen shot 2018-05-24 at 12 54 06 pm" src="https://user-images.githubusercontent.com/1587282/40500188-09e71a90-5f52-11e8-9ef8-63778119ae03.png">

Reply:
> Can we hide the Is this your site? prompt & buttons (.site-importer__site-importer-confirm-site-pane-container) until the image is fully loaded (or the Promise rejects)?

I like this. Updated the code to reflect that.